content/news/posts/2023/178-in-extended-support-phase.md

```diff
@@ -1,6 +1,7 @@
 ---
 layout: post
-title:  "PrestaShop 1.7.8 is now a LTS"
+aliases: ['/news/2023/178-lts-80/']
+title:  "PrestaShop 1.7.8 is now in the extended support phase"
```

Comment:
```suggestion
title:  "PrestaShop 1.7.8 is now in the extended support phase (LTS)"
```
Maybe the term is good to have, and it will allow to search on it too (as it's a common name used in dev environments).

Reply:
No no, we are changing this because "LTS" and "extended support" are not the same thing. LTS creates the expectation of bug fixes, whereas ES only brings limited support (critical issues only)